### PR TITLE
MFTF: Replace incorrect URLs in Tests and ActionGroups

### PR DESCRIPTION
**Pull request by lbajsarowicz**
### Description (*)
Replaced hardcoded URLs in Tests and ActionGroups to Page references

### Fixed Issues (if relevant)
N/A

### Manual testing scenarios (*)
N/A

### Questions or comments
N/A

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by dmytro-ch — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-6543](https://jira.corp.magento.com/browse/ENGCOM-6543) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
QA not applicable

File: app/code/Magento/Catalog/Test/Mftf/Test/AdminCheckPaginationInStorefrontTest.xml [hunk 1]
```diff
@@ -115,7 +115,7 @@
         <click selector="{{CatalogProductsSection.resetFilter}}" stepKey="clickOnResetFilter"/>
         <waitForPageLoad stepKey="waitForPageToLoad3"/>
         <selectOption selector="{{AdminProductGridFilterSection.productPerPage}}" userInput="30" stepKey="selectPagePerView"/>
-        <wait stepKey="waitFroPageToLoad1" time="30"/>
+        <waitForPageLoad stepKey="waitFroPageToLoad1" time="30"/>
```
Comment (ihor-sviziev) [inline app/code/Magento/Catalog/Test/Mftf/Test/AdminCheckPaginationInStorefrontTest.xml]:
```suggestion
        <waitForPageLoad stepKey="waitForPageToLoad1" time="30"/>
```

Comment (lbajsarowicz) [reply app/code/Magento/Catalog/Test/Mftf/Test/AdminCheckPaginationInStorefrontTest.xml]: Also noticed that after code push.

**Comment by lbajsarowicz** [comment]
C'mon @magento-engcom-team - no expertise for such a cleanups? :cry:

**Pull request merged**